pyrogram/client/storage/memory_storage.py

```diff
@@ -168,72 +168,57 @@ def get_peer_by_phone_number(self, phone_number: str):
 
         return self._get_input_peer(*r)
 
-    @property
-    def peers_count(self):
+    def get_peers_count(self):
         return self.conn.execute(
             "SELECT COUNT(*) FROM peers"
         ).fetchone()[0]
 
-    def _get(self):
-        attr = inspect.stack()[1].function
+    def _get(self, attr):
 
         return self.conn.execute(
             "SELECT {} FROM sessions".format(attr)
         ).fetchone()[0]
 
-    def _set(self, value):
-        attr = inspect.stack()[1].function
+    def _set(self, attr, value):
 
         with self.lock, self.conn:
             self.conn.execute(
                 "UPDATE sessions SET {} = ?".format(attr),
                 (value,)
             )
 
-    @property
-    def dc_id(self):
-        return self._get()
+    def get_dc_id(self):
+        return self._get("dc_id")
```

Comment:
And you should have not touched the body of ALL these methods, why did you edit them?

Reply:
Ok, it's alright, I forgot the _get and _set methods are depending on the method names, which we are editing